src/components/Execution/index.jsx

```diff
@@ -62,6 +64,10 @@ export const Execution = ({ nodes }) => {
     async () => {
       let webhooks = await client.stackClient.fetchJSON('GET', '/jobs/triggers')
```

Comment:
This could be replaced by `const webhooks = await client.all('io.cozy.triggers')` : https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/TriggerCollection.js#L66